relay-event-normalization/src/normalize/span/description/sql/mod.rs

```diff
@@ -133,22 +144,38 @@ fn scrub_queries_inner(db_system: Option<&str>, string: &str) -> (Option<String>
 
     let mut string = Cow::from(string.trim());
 
-    for (regex, replacement) in [
-        (&COMMENTS, "\n"),
-        (&INLINE_COMMENTS, ""),
-        (&NORMALIZER_REGEX, "$pre%s"),
-        (&WHITESPACE, " "),
-        (&PARENS, "$pre$post"),
-        (&COLLAPSE_PLACEHOLDERS, "$pre%s$post"),
-        (&STRIP_QUOTES, "$entity_name"),
-        (&COLLAPSE_ENTITIES, "$entity_name"),
-        (&COLLAPSE_COLUMNS, "$pre..$post"),
-    ] {
-        let replaced = regex.replace_all(&string, replacement);
-        if let Cow::Owned(s) = replaced {
+    if let Cow::Owned(s) = COMMENTS.replace_all(&string, "\n") {
+        string = Cow::Owned(s);
+    }
+    if let Cow::Owned(s) = INLINE_COMMENTS.replace_all(&string, "") {
+        string = Cow::Owned(s);
+    }
+    if let Cow::Owned(s) = NORMALIZER_REGEX.replace_all(&string, "$pre%s") {
+        string = Cow::Owned(s);
+    }
+    if db_system == Some("mysql") {
+        if let Cow::Owned(s) = DOUBLE_QUOTED_STRING_REGEX.replace_all(&string, "%s") {
             string = Cow::Owned(s);
         }
     }
+    if let Cow::Owned(s) = WHITESPACE.replace_all(&string, " ") {
+        string = Cow::Owned(s);
+    }
+    if let Cow::Owned(s) = PARENS.replace_all(&string, "$pre$post") {
+        string = Cow::Owned(s);
+    }
+    if let Cow::Owned(s) = COLLAPSE_PLACEHOLDERS.replace_all(&string, "$pre%s$post") {
+        string = Cow::Owned(s);
+    }
+    if let Cow::Owned(s) = STRIP_QUOTES.replace_all(&string, "$entity_name") {
+        string = Cow::Owned(s);
+    }
+    if let Cow::Owned(s) = COLLAPSE_ENTITIES.replace_all(&string, "$entity_name") {
+        string = Cow::Owned(s);
+    }
+    if let Cow::Owned(s) = COLLAPSE_COLUMNS.replace_all(&string, "$pre..$post") {
+        string = Cow::Owned(s);
+    }
```

Comment:
This is more verbose, but also more in line with [our internal guidelines](https://www.notion.so/sentry/Relay-Code-and-Architecture-4fac481cf27741aba54d39594f1cae58?pvs=4#941ff156a7064236910ce1d1684cdb30).